Packs/ServiceNow/Integrations/ServiceNowv2/Classifiers/classifier-ServiceNow_Mirror_In.json

```diff
@@ -0,0 +1,268 @@
+{
+	"brands": null,
+	"defaultIncidentType": "",
+	"description": "",
+	"feed": false,
+	"fromServerVersion": "6.0.0",
```

Comment:
```suggestion
	"fromVersion": "6.0.0",
```

Reply:
It generates automatically, should I change?